src/engine/bank03.asm

```diff
@@ -3416,7 +3416,50 @@ Script_d827: ; d827 (3:5827)
 	run_command ScriptCommand_QuitScriptFully
 ; 0xd82d
 
-	INCROM $d82d, $d880
+	INCROM $d82d, $d834
+	
+AfterTutorialBattleScript: ; d834 (3:5834)
+        start_script
+        run_command ScriptCommand_PrintTextString
+        tx Text05eb
+        run_command ScriptCommand_PrintTextString
+        tx Text05ef
+        run_command ScriptCommand_CloseTextBox
+        run_command ScriptCommand_MoveActiveNPC
+        dw $5896
```

Comment:
This appears to be a pointer to some movement data at the beginning of an `INCROM` (the `INCROM` right below `NPCMovement_d894`). You could simply add a label to replace the hard-coded address, or if you're willing, you could disassemble the movement data while you're at it.

Reply:
Yeah it seems that the script_extractor.py does not autogenerate the NPC_Movement. I went ahead and added it manually and updated the reference. Please check NPCMovement_d896